### ISSUE TITLE
Upgrade multer

### ISSUE THREAD

**Issue by ksmithut** [APIDevTools/swagger-express-middleware#74]
Looking through the code, I noticed a [piece of code](https://github.com/BigstickCarpet/swagger-express-middleware/blob/master/lib/request-parser.js#L36) that was commented out because it was waiting for a bug to be fixed. It appears that bug is fixed. If there are tests that to integration tests around file uploading, I can submit a PR fixing this.

**Comment by BigstickCarpet** [comment]
Awesome!  Thanks for letting me know about that bug being fixed.

Yes, there are already tests that test the file-uploading functionality, so if you want to submit a PR, then here are the steps:

1. Verify that all tests pass successfully on your machine with the existing dependencies
2. Upgrade Multer
3. Verify that all tests still pass
4. Submit a PR

Thanks!

**Comment by ksmithut** [comment]
One of the tests fail out of the box without any edits. It just hangs forever. It fails on [this line](https://github.com/BigstickCarpet/swagger-express-middleware/blob/master/tests/specs/mock/query-resource.spec.js#L521). Maybe different OSs cause different file sizes or something? Unrelated to this issue I suppose. I attempted to just update the dependency and uncomment the code, import the libraries that the new code required, then ran the tests again. The code might need some more work in order to get all the tests passing again. I can take a look sometime Wednesday evening to get it to work.

**Comment by BigstickCarpet** [comment]
Ah yeah... I think I remember running into the same issue, where tests would fail because the byte-size of files was incorrect.  It turned out to be caused by different git settings on different OSes.  When the git repo was cloned on Windows, line endings in text files were being converted from LF to CRLF, and binary files were similarly being changed.  I eventually fiddled with my git settings enough to get it working, but I can't recall what all I had to change. :-/
